### DISCUSSION TITLE
Welcome to the Power Apps Design Patterns Discussions!

### DISCUSSION THREAD

**Discussion by hugoabernier** [pnp/powerapps-designtoolkit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewpilcher** [comment]
Hi Hugo and Team!
Love your work. Always seems like I am only getting my feet wet though it has been over a year now. Looking to get more involved as I grow and bring in others.
